MHVLib/Models/Generated/Categories/MHVActionPlanTasksApi.h

```diff
@@ -43,11 +43,10 @@ NS_ASSUME_NONNULL_BEGIN
 ///
 /// @param actionPlanTask 
 /// 
-///  code:200 message:"OK",
 ///  code:201 message:"Created",
 ///  code:0 message:"Error"
 ///
-/// @return NSObject*
+/// @return MHVActionPlanTaskInstanceV2*
```

Comment:
I know it's the generated code from that big template, but they really return void & have the completion that returns the result object or error.